windows/SyncCrypto/dllmain.cpp

```diff
@@ -0,0 +1,26 @@
+#include "resource.h"
+
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+
+extern "C" {
+#include "../../native_lib/DDGSyncCrypto.h"
+#include "DDGSyncCryptoSizes.h"
+}
+
+BOOL APIENTRY DllMain( HMODULE hModule,
+                       DWORD  ul_reason_for_call,
+                       LPVOID lpReserved
+                     )
+{
+    switch (ul_reason_for_call)
+    {
+    case DLL_PROCESS_ATTACH:
+    case DLL_THREAD_ATTACH:
+    case DLL_THREAD_DETACH:
+    case DLL_PROCESS_DETACH:
+        break;
+    }
+    return TRUE;
```

Comment:
There's no code here, do we need this file?

Reply:
I'm a noob here, but I though I need an entry point where I include everything I need.

Reply:
Not unless there's some very strange requirements to do so (e.g. initialising some variables statically to ensure they don't get stripped out by the optimiser).

Reply:
Cool, got rid of it without any issues.